README.md

```diff
@@ -11,3 +11,12 @@ End-to-end development of a scalable day trading system for SENG 468
 - [Test Run 1 Single User](./Documents/Project_Overview1.pdfDocuments\Project_Overview1.pdf) due Febuary 15th 10%
   
 - [Project Overview](./Documents/Project_Overview1.pdfDocuments\Project_Overview1.pdf) for Reference
+
```

Comment:
If we're doing this, I feel like we cannot achieve microservices design that we're looking for